modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -29,14 +37,23 @@ def show
 
         private
 
-        def set_submission_attributes
-          @appeal ||= AppealsApi::NoticeOfDisagreement.find_by(id: params[:nod_id])
-          raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @appeal
+        def set_notice_of_disagreement
+          @nod_id ||= params[:nod_id]
+          raise Common::Exceptions::ParameterMissing, 'nod_id' unless @nod_id
 
+          @notice_of_disagreement ||= AppealsApi::NoticeOfDisagreement.find_by(id: @nod_id)
+          raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @notice_of_disagreement
+        end
+
+        def set_veteran_identifier
```

Comment:
I thought we were skipping veteran validation since we'd be removing it as PII and didn't want to complicate the checking with "does veteran data exist in form_data or not" stuff.

Reply:
Haven't refactored this with the new direction yet